src/main/java/org/opensearch/knn/index/IndexUtil.java

```diff
@@ -35,7 +35,7 @@
 public class IndexUtil {
 
     private static final Version MINIMAL_SUPPORTED_VERSION_FOR_LUCENE_HNSW_FILTER = Version.V_2_4_0;
-    private static final Version MINIMAL_SUPPORTED_VERSION_FOR_IGNORE_UNMAPPED = Version.V_2_10_0;
+    private static final Version MINIMAL_SUPPORTED_VERSION_FOR_IGNORE_UNMAPPED = Version.V_2_11_0;
```

Comment:
does this required because we downported that to 2.x after the code freeze for 2.10? Seems that [original PR ](https://github.com/opensearch-project/k-NN/pull/1071) doesn't have a label with required version, could you please add it @ryanbogan ?

Reply:
Yes, that's exactly what happened.  Working on the PR now